build/NuSpecs/MUXControls-Nuget-FrameworkPackage.props

```diff
@@ -113,6 +114,15 @@
     </ItemGroup>
   </Target>
 
+  <!-- Workaround for https://github.com/Microsoft/microsoft-ui-xaml/issues/599. Exclude Compact.xbf which is not in this project -->
+  <Target Name="_FixXamlCompactXbfPackaging" BeforeTargets="_ComputeAppxPackagePayload">
```

Comment:
Can you elaborate on what this is doing? What is the problem and how is this condition fixing it?

Reply:
The description has some more details but this sentence confused me: "This just exclude the compact.xbf from packaging if it's matching with the MSBuildProjectName". I think this should have said if the app being packaged *doesn't match* MSBuildProjectName? 

---
In reply to: [278214089](https://github.com/Microsoft/microsoft-ui-xaml/pull/610#discussion_r278214089) [](ancestors = 278214089)